scene/gui/split_container.cpp

```diff
@@ -267,6 +284,9 @@ void SplitContainer::_resort() {
 		return;
 	}
 	dragging_area_control->set_visible(!collapsed);
+	if (touch_dragger_enabled) {
+		touch_dragger->set_visible(dragging_enabled);
+	}
```

Comment:
```suggestion
	if (touch_dragger_enabled) {
		touch_dragger->set_visible(dragging_enabled && dragger_visibility == DRAGGER_VISIBLE);
	}
```
I guess the touch dragger should use dragger_visibility since the regular grabber is replaced?
I'm not sure if it should follow auto_hide though, since it should always be visible.

Also ideally this wouldn't be in _resort, it can be updated when related properties are updated. But the dragging_area_control already does stuff like this here, so you can leave it here for now if you want.